patterns/1-initial/good-first-project.md

```diff
@@ -0,0 +1,88 @@
+    - _Ease of contribution_ -> Are the required ancillary resources set up and is the code modular enough to facilitate contribution?
+3. __People Compatibility__
+    - _Team Readiness for InnerSource_ -> Are the team members aware of and ready for the challenges of maintaining an InnerSource project?
+    - _Management Support_ -> Is the management willing to support the developers working on the project and the initiative in general?
+
+As one specific implementation of the pattern, we present a tool for estimating the _fitness_ of a candidate project along the six dimensions, and also present an an aggregate score which is a weighted sum of the scores for the dimensions (weights are determined by how many people voted a characteristic as important in the [2020 InnerSource Survey](https://tapjdey.github.io/InnerSource_Survey_2020/index.html)).
+The tool is written in __R__, and can be run from the [GitHub repository](https://github.com/tapjdey/InnerSource_Project_Fitness) (recommended - instructions in the repository) or from an [online instance](https://tapjdey.shinyapps.io/InnerSource_Fitness_Checker/). The tool is available in Chinese and English (language can be changed on the fly).
+
+## Resulting Context
+
+In general, application of the framework should help assess the InnerSource-readiness of individual projects.
+
+The _fitness score_ of a project can be used as an indication about its suitability for InnerSource style development, which can help in selecting the most appropriate candidate project(s).
+
+The questionnaire presented in the tool also serves as a de-facto guide as to which aspects need attention for making a project better suited for InnerSource style development.
+
+## Rationale
+
+The two most important works in this context are by [Klaas-Jan Stol and Brian Fitzgerald](https://ulir.ul.ie/bitstream/handle/10344/4443/Stol_2014_inner.pdf?sequence=2) and [Silona Bonewald](https://innersourcecommons.org/assets/files/InnerSourceChecklist.pdf).
+
+In their work, _Stol and Fitzgerald_ presented a set of nine key factors that are important for organizations to consider when adopting inner source. However, that model looked at an organization as a whole, and while a number of organizational characteristics are applicable to an individual project, some aspects are not, and some project characteristics are unaccounted for in the model. They presented three main factors: Software Product, Practices and Tools, People and Management, and three sub-factors on each of them.
+
+_Silona Bonewald_, on the other hand, presented a near-exhaustive checklist about different aspects to consider while adopting InnerSource at an organization. However, not all of them are applicable while considering an individual project. Ideas from the two works described earlier are combined in framing the pattern presented here.
+
+## Known Instances (optional)
```

Comment:
```suggestion
## Known Instances

TBD
```

Even though this section is optional, I would keep it in.
Finding known instances of a pattern is the next step to improving the pattern further and eventually leveling it up to the next maturity level (Structured).

Showing an empty section here might serve as motivation for others to contribute their "known instance".

Reply:
Sounds great!